### ISSUE TITLE
Over-Assigning Damage on Assets

### ISSUE THREAD

**Issue by Malacante** [halogenandtoast/ArkhamHorror#97]
It is possible to assign an asset more damage/horror than the amount that would defeat it, as long as that damage/horror is assigned simultaneously. For example, if I have a Bulletproof Vest (4 health) with three damage assigned, and take a 2 damage attack, I can assign both of those damage to the Bulletproof Vest

**Comment by halogenandtoast** [comment]
> An asset cannot be assigned damage beyond the amount of damage it would take to defeat the card, and cannot be assigned horror beyond the amount of horror it would take to defeat the card.

I interpreted this as it would be impossible to assign both damage in this case.

**Comment by halogenandtoast** [comment]
Going to close this since I believe the rule is captured correctly here. If wrong feel free to reopen and link the rule.

**Comment by Malacante** [comment]
Your understanding of the rules is correct; I wasn't clear. I was describing what I was able to do in-game, in violation of the above rule. As another example, I took an attack from Umhordoth and was able to assign all three damage and horror to a Peter Sylvester (2) with 2 horror already on him. I shouldn't be able to do this. On that occasion, the game did fight me a bit - after assigning some of the damage/horror over his printed values, it seemed to undo it and the amount on him went down to his printed values. However, I just kept trying to assign the damage/horror, and eventually it gave in and let me defeat Peter without putting anything on my other cards/investigator. I'll see if this is still an issue on the latest build.
